frontend/packages/container-security/locales/en/container-security.json

```diff
@@ -1,37 +1,36 @@
 {
-  "Image Manifest Vuln Details": "Image Manifest Vuln Details",
+  "Image Manifest Vulnerabilities details": "Image Manifest Vulnerabilities details",
   "{{total, number}} total": "{{total, number}} total",
   "Quay Security Scanner has detected {{total, number}} vulnerabilities.": "Quay Security Scanner has detected {{total, number}} vulnerabilities.",
   "Patches are available for {{fixableCount, number}} vulnerabilities.": "Patches are available for {{fixableCount, number}} vulnerabilities.",
   "{{title}} vulnerabilities": "{{title}} vulnerabilities",
   "Registry": "Registry",
   "Manifest": "Manifest",
   "Affected Pods": "Affected Pods",
-  "Image Name": "Image Name",
+  "Image name": "Image name",
   "Namespace": "Namespace",
-  "Highest Severity": "Highest Severity",
+  "Highest severity": "Highest severity",
   "Fixable": "Fixable",
   "Total": "Total",
-  "No Image Vulnerabilities found": "No Image Vulnerabilities found",
+  "No image vulnerabilities found": "No image vulnerabilities found",
   "Image Manifest Vulnerabilities": "Image Manifest Vulnerabilities",
   "Container": "Container",
   "Image": "Image",
-  "Security Scan": "Security Scan",
+  "Security scan": "Security scan",
   "No vulnerabilities found": "No vulnerabilities found",
   "Severity": "Severity",
   "Vulnerabilities": "Vulnerabilities",
-  "No Image Vulnerabilities Found": "No Image Vulnerabilities Found",
   "Vulnerability": "Vulnerability",
   "Package": "Package",
-  "Current Version": "Current Version",
-  "Fixed in Version": "Fixed in Version",
+  "Current version": "Current version",
+  "Fixed in version": "Fixed in version",
   "{{fixableCount, number}} of {{totalCount, number}} fixable": "{{fixableCount, number}} of {{totalCount, number}} fixable",
   "This project's container images from Quay are analyzed to identify vulnerabilities. Images from other registries are not scanned.": "This project's container images from Quay are analyzed to identify vulnerabilities. Images from other registries are not scanned.",
   "Container images from Quay are analyzed to identify vulnerabilities. Images from other registries are not scanned.": "Container images from Quay are analyzed to identify vulnerabilities. Images from other registries are not scanned.",
-  "Vulnerable Container Images": "Vulnerable Container Images",
+  "Vulnerable container images": "Vulnerable container images",
   "View all": "View all",
   "{{vulnImageCount, number}} total": "{{vulnImageCount, number}} total",
-  "Fixable Container Images": "Fixable Container Images",
+  "Fixable container images": "Fixable container images",
```

Comment:
It looks like `container` is a proper noun too so `container` and `image` can be title case.